microhapulator/__init__.py

```diff
@@ -10,11 +10,10 @@
 # Core libraries
 import builtins
 from gzip import open as gzopen
-from sys import stdin, stdout, stderr
+import sys
```

Comment:
It turns out, if we import module members, this can lead to problems with pytest capturing stdout/stderr. In this case it's best to import the module and access members using the fully qualified name, including the module.